packages/apps/spaces/app/customer-map/src/graphql/arrBreakdown.generated.ts

```diff
@@ -68,94 +41,53 @@ export const ArrBreakdownDocument = `
     `;
 
 export const useArrBreakdownQuery = <
-  TData = ArrBreakdownQuery,
-  TError = unknown,
->(
-  client: GraphQLClient,
-  variables?: ArrBreakdownQueryVariables,
-  options?: Omit<
-    UseQueryOptions<ArrBreakdownQuery, TError, TData>,
-    'queryKey'
-  > & {
-    queryKey?: UseQueryOptions<ArrBreakdownQuery, TError, TData>['queryKey'];
-  },
-  headers?: RequestInit['headers'],
-) => {
-  return useQuery<ArrBreakdownQuery, TError, TData>({
-    queryKey:
-      variables === undefined ? ['ArrBreakdown'] : ['ArrBreakdown', variables],
-    queryFn: fetcher<ArrBreakdownQuery, ArrBreakdownQueryVariables>(
-      client,
-      ArrBreakdownDocument,
-      variables,
-      headers,
-    ),
-    ...options,
-  });
-};
+      TData = ArrBreakdownQuery,
```

Comment:
Is there a problem with your prettier? This seems odd compared with other generated files.

Reply:
regenerated and pushed, no idea why it was reformatted this way